src/F3DAnimationManager.cxx

```diff
@@ -85,7 +85,7 @@ void F3DAnimationManager::Initialize(const F3DOptions& options, vtkImporter* imp
     F3DLog::Print(F3DLog::Severity::Info, "\n");
   }
 
-  if (options.AnimationIndex >= 0)
+  if (options.AnimationIndex > 0)
```

Comment:
You sure about this, there is no if for animationIndex == 0 ?

Reply:
Yes because 0 is the default now.

Reply:
ok, then you need to check that we only enable AnimationIndex 0 if availAnimation >= 1.

Reply:
I've rework this block, let me know if it looks better.

Reply:
yup, better.